include/libopencm3/stm32/f7/rcc.h

```diff
@@ -571,6 +571,7 @@
 
 /* --- RCC_DCKCFGR2 values -------------------------------------------------- */
 
+#define RCC_DCKCFGR2_DSISEL			(1<<28)
```

Comment:
what f7 part are you looking at here? I can't find that in the ref mans I'm skimming.

Reply:
RM0410 has DSISEL on bit 30 at least?

Reply:
sorry, this is a bug which is fixed in just the next commit (stm32f4-7: merge/synchronize rcc between f4 and f7)

Reply:
ah crap! this is a bug which is fixed in just the next commit

Reply:
please don't add it in wrongly in this one at all.